packages/helium-cli/emissions/mobile.json

```diff
@@ -1,14 +1,14 @@
 [
   {
-    "startUnixTime": 1679064129,
-    "emissionsPerEpoch": "144657534246575"
+    "startTime": "2023-04-18T00:00:00Z",
+    "emissionsPerEpoch": "109,392,265.193370"
```

Comment:
* `109,392,265.193370 * (60/66) = 99,447,513.81 MOBILE` a day for Hotspots
* `109,392,265.193370 * ( 6/66) = 9,944,751.38 MOBILE` a day for veHNT delegators

Only 66% of emissions are expected until other parties like mappers (10%), SPs (20%), and Oracles (4%) are online and rewardable. rest of the